apps/journeys-admin/src/components/JourneyVisitorsList/VisitorCard/VisitorCardDetails/VisitorCardDetails.tsx

```diff
@@ -0,0 +1,147 @@
+import { ReactElement } from 'react'
+import Stack from '@mui/material/Stack'
+import { useTranslation } from 'react-i18next'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import { format, parseISO } from 'date-fns'
+import Skeleton from '@mui/material/Skeleton'
+import { GetJourneyVisitors_visitors_edges_node_events as Event } from '../../../../../__generated__/GetJourneyVisitors'
+
+interface Props {
+  name?: string
+  events: Event[]
+  loading: boolean
+}
+
+export function VisitorCardDetails({
+  name,
+  events,
+  loading
+}: Props): ReactElement {
+  const { t } = useTranslation('apps-journeys-admin')
+
+  const eventsFilter: Array<Event['__typename']> = [
+    'ChatOpenEvent',
+    'TextResponseSubmissionEvent',
+    'RadioQuestionSubmissionEvent'
+  ]
+
+  const filteredEvents = events.filter((event) =>
+    eventsFilter.includes(event.__typename)
+  )
+
+  return (
+    <>
+      <Box sx={{ pt: 3 }}>
+        <DetailsRow label={t('Name')} value={name} loading={loading} />
+      </Box>
+
+      {filteredEvents.map((event) => {
+        if (event.__typename === 'ChatOpenEvent') {
+          return (
+            <DetailsRow
+              key={event.id}
+              label={t('Chat Started')}
+              value={format(parseISO(event.createdAt), 'h:mmaaa')}
+              chatEvent
+              loading={loading}
+            />
+          )
+        } else {
+          return (
+            <DetailsRow
+              key={event.id}
+              label={event.label}
+              value={event.value}
+              loading={loading}
+            />
+          )
+        }
+      })}
+    </>
+  )
+}
+
+interface DetailsRowProps {
+  label: string | null
+  value?: string | null
+  chatEvent?: boolean
+  loading: boolean
+}
+
+function DetailsRow({
+  label,
+  value,
+  chatEvent = false,
+  loading
+}: DetailsRowProps): ReactElement {
+  const textColor = chatEvent ? 'primary' : 'secondary'
+  const isBold = chatEvent ? 900 : 'normal'
+  const variant = chatEvent ? 'subtitle1' : 'body2'
```

Comment:
Try subtitle2? Subtitle 1 is a bit too much
```suggestion
  const variant = chatEvent ? 'subtitle2' : 'body2'
```